### ISSUE TITLE
aws_dynamodb_table fails in govcloud after BillingMode support added in #6648 

### ISSUE THREAD

**Issue by briceburg** [terraform-providers/terraform-provider-aws#6919]
We are unable to mange `aws_dynamodb_table` resources while terraforming in govcloud after BillingMode support from #6648 was merged. My guess is the dynamodb implementation in govcloud does not yet support BillingMode and the API is blowing up.

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

```
Terraform v0.11.10
+ provider.aws v1.52.0
```

### Affected Resource(s)

* aws_dynamodb_table

### Actual Behavior

```
* aws_dynamodb_table.tfstate: ValidationException: One or more parameter values were invalid: Unsupported input parameter BillingMode
```

### Steps to Reproduce

1. `AWS_PROFILE=govcloud terraform apply`

### References

* https://github.com/terraform-providers/terraform-provider-aws/pull/6648#pullrequestreview-186388856

**Comment by bflad** [comment]
Fix submitted: #7064

**Comment by bflad** [comment]
The fix for this has been merged and will release with version 1.55.0 of the AWS provider, likely in the next two days.

**Comment by bflad** [comment]
This has been released in [version 1.55.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1550-january-10-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
